src/main/java/duke/command/ToDoCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.ToDo;
+import duke.ui.Ui;
+
+/**
+ * Is responsible for handing commands starting with <code>todo</code>.
```

Comment:
Maybe it should start with a verb?

Reply:
I think "is" is a verb

Reply:
I have updated my code based on your reviews. Thank you for spending time looking through this!